### PR TITLE
docs: add data pipeline in faq and troubleshooting

### PR DESCRIPTION
**Pull request by zxkane**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

(describe what this merge request does)

## Implementation highlights

(describe how the merge request does for feature changes, share the RFC link if it has)

## Test checklist

- [ ] add new test cases
- [ ] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [ ] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [ ] deploy with reporting

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

File: docs/en/faq.md [hunk 1]
```diff
@@ -2,34 +2,90 @@
...
 ## General
 
-**Q:  What is {{ solution_name }}?**<br>
+### What is {{ solution_name }}?
+
 An AWS Solution that enables customers to build clickstream analytic system on AWS easily. This solution automates the data pipeline creation per customers' configurations with a visual pipeline builder, and provides SDKs for web and mobiles apps (including iOS, Android, and Web JS) to help customers to collect and ingest client-side data into the data pipeline on AWS. After data ingestion, the solution allows customers to further enrich and model the event data for business users to query, and provides built-in visualizations (e.g., acquisition, engagement, retention) to help them generate insights faster.
 
+## Data pipeline
+
+### How do I choose the Redshift type for data modeling?
```
Comment (rrxie) [inline docs/en/faq.md:11]: 这里讲的更像是选择 Redshift 的条件？

When do I choose Redshift for data modeling?

File: docs/en/faq.md [hunk 2]
```diff
@@ -2,34 +2,90 @@
...
+- The data processing interval is equal to or larger than one hour.
+- The reporting querying and other usage, such as ETL, is intensive for a few hours at most.
+- Redshift is not used for streaming ingestion.
+- The estimated cost is lower than the provisioned cluster.
+
+### How do I monitor the health of the data pipeline for my project?
+
+You can open the built-in [observability dashboard][monitoring-dashboard] to view the key metrics of your data pipeline.
+
+The dashboard displays metrics for different components of your data pipeline, including ingest, process, and modeling.
```
Comment (rrxie) [inline docs/en/faq.md:24]: including data ingestion, processing, and modeling.

File: docs/en/faq.md [hunk 3]
```diff
@@ -2,34 +2,90 @@
...
+
+- **Data Ingestion - Server**:
+    - **Server Request Counts**: The total requests received by the ingestion servers in the given period. You can use it to calculate the request per second (RPS) of the ingestion servers.
+    - **Server Response Time**: The average response time in seconds of the ingestion servers in the given period.
+    - **Server(ECS) Tasks** The number of tasks/instances running for the ingestion servers.
+- **Data Ingestion - Sink - Kinesis Data Stream** (available when enabling KDS as ingestion sink):
+    - **Kinesis Throttled and Failed Records**: The total putting records of KDS were throttled or failed in the given period. The lower value is better.
```
Comment (rrxie) [inline docs/en/faq.md:31]: The lower the value is, the better.

这里的数值是否有个范围？

**Review by rrxie — COMMENTED** [review]
commented.

File: docs/en/faq.md (hunk 1 above)
Comment (zxkane) [reply docs/en/faq.md:11]: Redshift has two types with different cost models: provisioned and serverless. It's talking about when to use Redshift serverless.

File: docs/en/faq.md (hunk 3 above)
Comment (zxkane) [reply docs/en/faq.md:31]: Any value higher than 0 means something wrong happened.

**Review by rrxie — DISMISSED** [review]
Approved

**Review by rrxie — APPROVED** [review]

**Pull request merged**